lib/beaker/hypervisor/openstack.rb

```diff
@@ -220,16 +220,15 @@ def provision
 
       @hosts.each do |host|
         ip = get_ip
-        host[:vmhostname] = ip.ip.gsub '.','-'
-        host[:vmfqdn] = host[:vmhostname] + '.rfc1918.puppetlabs.net'
```

Comment:
I'm not sure if this is safe to remove, but it looks like `:vmfqdn` is only used within this openstack hypervisor, so as long as no one out in the wild has written any code that relies on this, we should be good to go. @lucywyman @johnduarte any thoughts on the possibility someone has written some code reliant on `:vmfqdn`?